scripts/openmc-update-inputs

```diff
@@ -247,6 +254,9 @@ def update_materials(root):
             if 'name' in nuclide.attrib:
                 nucname = nuclide.attrib['name']
                 nucname = nucname.replace('-', '')
+                s = re.search("\s*(?<![A-z])\d{4,6}\s*", nucname)
+                if s:
+                    nucname = replace_zaid(s.group(0))
```

Comment:
This is probably overkill. I'd suggest:
```Python
# If name looks like a number (ZAID), convert it to a proper nuclide name
if nucname.strip().isnumeric():
    return openmc.data.neutron._get_metadata(int(nucname))[0]
```
You could get rid of the `replace_zaid` function then too.